.clang-format

```
@@ -0,0 +1,117 @@
+Language: Cpp
+BasedOnStyle: LLVM
+AccessModifierOffset: -2
+AlignAfterOpenBracket: Align
+AlignConsecutiveAssignments: true
+AlignConsecutiveDeclarations: true
+AlignEscapedNewlines: Left
+AlignOperands:   true
+AlignTrailingComments: true
+AllowAllParametersOfDeclarationOnNextLine: false
+AllowShortBlocksOnASingleLine: true
+AllowShortCaseLabelsOnASingleLine: true
+AllowShortFunctionsOnASingleLine: All
+AllowShortIfStatementsOnASingleLine: true
+AllowShortLoopsOnASingleLine: false
+AlwaysBreakAfterDefinitionReturnType: None
+AlwaysBreakAfterReturnType: AllDefinitions
+AlwaysBreakBeforeMultilineStrings: false
+AlwaysBreakTemplateDeclarations: true
+BinPackArguments: true
```

```suggestion
BinPackArguments: false
```

I'd disable this so we have
```
	table_grid.setLeafSize(cfg_table_downsample_leaf_size_,
	                       cfg_table_downsample_leaf_size_,
	                       cfg_table_downsample_leaf_size_);
```
instead of
```
	table_grid.setLeafSize(cfg_table_downsample_leaf_size_, cfg_table_downsample_leaf_size_,
	                       cfg_table_downsample_leaf_size_);
```